content/projects/tdd/simple-calculator-part1/_index.md

```diff
@@ -4,33 +4,88 @@ pre: "<b>EASY: </b>"
 ready: true
 ---
 
-We'll build a calculator to do some basic arithmetic. Please follow the instructions below. Please take all the naming conventions in this document with a gran of salt. There are lots and lots of test frameworks in existence, each with its own conventions. Please follow whatever conventions are set out by the test tool that you are using.
+The objective of this project is to build a calulator that can perform multiplication and addition equations on multiple integers. Do not build a Front-end(UI). Complete this project by using a TDD approach. 
 
-Also, please don't build a frontend.
+The basic TDD approach is as follows:
 
-We'll follow a TDD approach. The basic approach is
+1. RED: Write tests. It should fail initially because there isn't any code that it is testing.
+2. GREEN: Write code to make the tests pass
+3. REFACTOR: Make sure code is understandable and clean.
 
-1. RED: we will write ONE failing test
-2. GREEN: we will make sure our test (and all tests written so far) pass
-3. REFACTOR: if our code is untidy we will clean it up
-4. repeat all the steps until all the functionality we need has been built and tested
+Remember to make sure your tests still pass after refactoring it.
 
-## 1. Make a function that adds two numbers
+## Set up environment
 
-First write a test function. it can be called `test_function` or something similar. It needs to make sure that the `add` function works. (of course we haven't written `add` yet. We'll write our tests first).
+### JavaScript:
 
-- write a test to demonstrate that `add(0,0)` will return 0. Run the test. It should fail. Now make the test pass by creating an `add` function.
-- write a test to demonstrate that `add(-1,-1)` should return -2 then edit `add` to make your test pass
-- write a test to demonstrate that `add(4,5)` should return 9 then edit `add` to make the test pass
+Use Jasmine to test your code. *Please do not use the SpecRunner* to test your code. Run Jasmine on the terminal.
+- [Running Jasmine on the Terminal](../../../topics/jasmine-unit-tests/_index.md). Look under the heading: *Getting set up (like a boss)* for instructions to set up.
 
-## 2. Make sure you can add as many numbers as you want
+After setting up Jasmine on the terminal, please ensure that your directory has the following:
 
-Now write a test to make sure you can add a bunch of numbers. Eg `add(1,2,3,4)` should return 10. Think of the tests yourself
+- A `src` folder with your source file : `simpleCalculator.js` .
```

Comment:
@sheenarbw  @Ferronique-May 

Note: Jasmine spec files must be named as *spec.js so in this case "simple-calculator-spec.js". Otherwise the tests will not run. 

See usage heading on Jasmine page: https://www.npmjs.com/package/jasmine-node